plugins/registry.ts

```diff
@@ -187,20 +187,19 @@ export default class PluginRegistry {
     // Add a button to the channel intro message.
     // Accepts the following:
     // - icon - React element to use as the button's icon
-    // - action - a function called when the button is clicked, passed the channel and channel member as arguments
-    // - tooltipText - string or React element shown for tooltip appear on hover
+    // - action - a function called when the button is clicked
+    // - text - a localized string to use as the button's text
```

Comment:
@sbishel We never actually passed any arguments into `action` before, so I just removed the mention it from this comment. If you'd prefer, I can add those back in and have the arguments passed in though like we previously wanted to do.

I also removed `tooltipText` since we never actually used it

Reply:
Yeah, I think we ended up using the "currentChannel" and therefore didn't need the arguments passed. Other plugins may need/want arguments, but from boards side, we don't need them.

Reply:
This is no longer needed since I just renamed `tooltipText` to `text` instead of adding a second field